core/client/fs/src/main/java/alluxio/client/meta/RetryHandlingMetaMasterConfigClient.java

```diff
@@ -63,8 +64,13 @@ protected void afterConnect() {
   }
 
   @Override
-  public List<ConfigProperty> getConfiguration() throws IOException {
-    return retryRPC(() -> mClient.getConfiguration(GetConfigurationPOptions.getDefaultInstance())
-        .getConfigsList());
+  public GetConfigurationPResponse getConfiguration() throws IOException {
```

Comment:
Could we return a non proto object here? We try to keep all the proto responses limited to the client wrappers.

Reply:
done